### PR TITLE
try upgrading to React 16 to suss out the issues

### PR DESCRIPTION
**Pull request by kevinbarabash**
There are three failing errors that I wasn't able to resolve...yet.  Two of them stem from our use of `findDOMNode` and the third is `Enzyme Internal Error: unknown node with tag 12`.  We'll to disable that test until https://github.com/airbnb/enzyme/issues/1620 is fixed.

### CODE REVIEW

File: packages/wonder-blocks-core/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "peerDependencies": {
     "aphrodite": "^1.2.5",
-    "react": "^15.3.2",
+    "react": "^16.4.1",
```
Comment (kevinbarabash) [inline packages/wonder-blocks-core/package.json]: It took me a while for me figure out that these needed to be updated as well.

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 03638b1f09febead0f15cd6ec8ee229ffd38b10e

https://deploy-preview-162--wonder-blocks.netlify.com

**Comment by kevinbarabash** [comment]
@somewhatabstract any thoughts about the `findDOMNode` related failures.

**Comment by somewhatabstract** [comment]
@kevinbarabash The issue, at least for the modal test failures I saw, is that the nodes haven't been mounted at the point `findDOMNode` is called and so there are no DOM nodes to speak of. Those tests probably need re-writing to be similar to the ones I wrote for the tooltip components so that they mount first and get real mounted nodes to use with `findDOMNode`.

**Comment by somewhatabstract** [comment]
I can take a closer look a little later today to probably delve into them, if you'd like.

**Comment by kevinbarabash** [comment]
> Those tests probably need re-writing to be similar to the ones I wrote for the tooltip components so that they mount first and get real mounted nodes to use with `findDOMNode`.

I might give that a shot this weekend.

**Comment by wishdasher** [comment]
I'm going to rebase and push

**Pull request merged**